plugins/jobs/listBuilds.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const boom = require('boom');
+const hoek = require('hoek');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const listSchema = joi.array().items(schema.models.build.get).label('List of builds');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/jobs/{id}/builds',
+    config: {
+        description: 'Get builds for a given job',
+        notes: 'Returns builds for a given job',
+        tags: ['api', 'jobs', 'builds'],
+        handler: (request, reply) => {
+            const factory = request.server.app.jobFactory;
+
+            return factory.get(request.params.id)
+              .then(job => {
+                  if (!job) {
+                      throw boom.notFound('Job does not exist');
+                  }
+
+                  let config = {};
+
+                  if (Object.keys(request.query).length !== 0) {
+                      config = {
+                          paginate: hoek.applyToDefaults({}, {
+                              page: request.query.page,
+                              count: request.query.count
+                          })
+                      };
+
+                      if (request.query.sort) {
+                          config.sort = request.query.sort;
+                      }
+                  }
+
+                  return job.getBuilds(config);
+              })
+              .then(builds => reply(builds.map(b => b.toJson())))
+              .catch(err => reply(boom.wrap(err)));
+        },
+        response: {
+            schema: listSchema
+        },
+        validate: {
+            query: {
+                page: joi.number().integer().positive(),
```

Comment:
Yes, but it was exposed as an object with page and count. You can't access the page or count individually. I modified it so it has sort in there: https://github.com/screwdriver-cd/data-schema/blob/master/api/pagination.js so I'm just gonna change it to use that now